numpy/core/numeric.py

```diff
@@ -1581,6 +1581,10 @@ def cross(a, b, axisa=-1, axisb=-1, axisc=-1, axis=None):
         axisa, axisb, axisc = (axis,) * 3
     a = asarray(a)
     b = asarray(b)
+
+    if (a.ndim < 1) or (b.ndim < 1):
+        raise Exception("At least one array has zero dimension")
```

Comment:
If you want to catch a ValueError, you need to use ValueError as an exception. Also, there is still a grammar error in the message.
```suggestion
        raise ValueError("At least one array has zero dimensions")
```